bevm/api.go

```diff
@@ -664,6 +664,74 @@ func decodeEvmArray(abiType string, abi abi.Argument, arg interface{}) (
 	return arr.Interface(), nil
 }
 
+// EncodeEvmResult encodes an EVM call result into JSON.
+// This can be useful for command-line tools or calls serialized over the
+// network.
+func EncodeEvmResult(result interface{}, outputs abi.Arguments) (string, error) {
```

Comment:
Just for my readability - chose 0-2 of the following points:
- `s/result/value/`
- re-arrange to have the `result/value` always at 2nd argument (preferred) or 1st argument in `[Ee]ncodeEvmResult`

Reply:
Actually, I chose this naming/order in symmetry with `DecodeEvmArgs()` / `decodeEvmValue()`...
The rationale being that the public functions use "Args" or "Result" indicating the direction of the conversion, and the private lower-level functions use "value".

Reply:
Hmm - but if I read the method here and see `result`, I think that this will hold the result of the method.

Can you at least make the comment more clear that `result` is the input parameter here? The first few times I read the comment I didn't understand it.

Reply:
I guess different people, different logic ;-)
I renamed functions and variables to use the term "return value" instead of "result" to improve clarity.

Reply:
still confusing to me. Anyway...